_posts/2018-05-18-focused-workspaces.md

```diff
@@ -0,0 +1,136 @@
+---
+layout     : post
+title      : "Ease the Transition to a Monorepo with Focused Workspaces"
+author     : Bryan Wain
+author_url : "https://github.com/bdwain"
+date       : 2018-05-18 00:00:00
+categories : announcements
+share_text : "Ease the Transition to a Monorepo with Focused Workspaces"
+---
+
+In a [previous post](/blog/2017/08/02/introducing-workspaces/), we wrote about monorepos and how Yarn Workspaces makes working with them much simpler. Unfortunately, moving to a monorepo is not always an easy choice because without the right tooling, they can often harm the developer exerpience instead of help it.
+
+## The Problem With Monorepos
+
+One of the main reasons projects usually move to a monorepo is because they make it much easier to make changes across multiple packages by allowing you make those changes in a single pull request. One of the ways Yarn Workspaces helps improves this workflow is by automatically symlinking sibling package dependencies in the node_modules folder, allowing you to immediately see the results of a change to one package when working on another.
```

Comment:
Yarn Workspaces helps improves this => Yarn Workspaces improves this

Reply:
fixed